packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -83,7 +83,7 @@ export default {
   },
   computed: {
     isPicture() {
-      return typeof this.src === "object";
+      return Object.keys(this.src).length > 0 && typeof this.src === "object";
```

Comment:
@justyna-13 IMO first should you check `typeof this.src === "object"` and after that do other test :) WDYT about this fix https://github.com/DivanteLtd/storefront-ui/pull/1167

Reply:
Ok so I'm closing this PR :)